### DISCUSSION TITLE
Higher resolution of icons, fixed size of main window, problem with icons of files on encrypted partition

### DISCUSSION THREAD

**Discussion by daniel12838** [Hofknecht/SystemTrayMenu#502] [Ideas]
Hi.

I use SystemTrayMenu since today and I want to say that it's great.
It's really helpful program. Thank you for creating and releasing it!

I have 2 ideas and 1 problem.

- I noticed that big icons are low resolution. Maybe it's intentional for better performance? https://i.imgur.com/kJ0G1Wc.png
- Minimum width and minimum height would be great. It would enable to set fixed size of main window.
- I have small problem with icons of apps which are located on my encrypted E:\ partition. It happened after reboot after first using SystemTrayMenu. These apps work, just no icons. https://i.imgur.com/TmVMr2u.png

**Comment by Hofknecht** [comment]
Hey @daniel12838 
Thanks for the feedack!
Thanks for the suggestion!
We'll look at it again later and probably only implement it in v2.

1.
![image](https://github.com/Hofknecht/SystemTrayMenu/assets/52528841/071c35ee-c82d-458f-b5b8-e4debf611a44)
If you set the iconsize to 200% it is currently not displayed in a better resolution, we know this issue. The current native method we use doesn't work as far as I've tried in higgher resolution. But there are other example projects where you can get the icons with even higher resolution. So should be possible to improve. This needs to be analyzed more closely in the future. A problem for me was that it should look exactly the same at 100% and 200%, e.g. In the normal folder, the icon at 200% is not just bigger but looks completely different and not in the same way as in Windows Explorer. But we will look at it again in the future.

2.
i agree, Minimum width and minimum height would be great. 
We've had this idea before, so far it was only half necessary but actually a very nice feature.
We often wait until someone mentions it : )

3. 
"Encrypted icons not shown". That topic is new for us. At my workplace harddrive is encrypted and there it works. we need to reproduce this. I'm not sure at the moment how best to reproduce. Did this always happen or just once? so windows explorer can display the icons correctly? And even after restarting the problem persists? Even if you copy a new file with an icon? I think an encrypted hard drive shouldn't be noticeable to the user in Windows or a app in windows. the files and icons are of course normally readable for programs. Possibly authorization problems? are you an admin and can read and write there? I'm not sure where this comes from, but if you have any more information that will help us

**Comment by daniel12838** [comment]
Thank you for answer! :)

3.
I think I found reason why it happens in my case.
It depends order running SystemTrayMenu and mounting encrypted partition. 
If SystemTrayMenu runs before encrypted partition is mounted, there are no icons.
If SystemTrayMenu runs after encrypted partition is mounted, icons are ok.

In my case: 
SystemTrayMenu is on uncrypted C:\ and it auto runs with Windows before encrypted partition E:\ is mounted.
Right clicking tray icon and clicking "Restart" helps.

**Comment by Hofknecht** [comment]
thank you for the feedback!
ok we need to think about that
